### DISCUSSION TITLE
How to use translation in a stateless widget?

### DISCUSSION THREAD

**Discussion by jeremyhalin** [nylo-core/nylo#31] [Q&A] [answered]
I was wondering if it was possible? Of course, I thought about passing props, but is it the right way?

**Comment by agordn52** [comment, marked as answer]
Hi @jeremyhalin,

I'd recommend checking out the docs here https://nylo.dev/docs/3.x/localization

Hope that helps 👍

**Reply by jeremyhalin** [reply]
Of course, I've read the docs before asking. But nothing about using localization in a stateless widget, is there a way to call `trans` outside a stateful widget?

**Reply by agordn52** [reply]
Ohh ok.

Where are you trying to use the `trans` helper in your code? In a class method?

**Reply by jeremyhalin** [reply]
For example, in the `build` method of a `StatelessWidget` (sorry for late response).

**Reply by agordn52** [reply]
Hi @jeremyhalin,

You can in this example here https://github.com/nylo-core/nylo/blob/3.x/lib/resources/pages/home_page.dart

Check out line 85, it'll work the same for stateless widgets.

Hope that helps.

**Reply by jeremyhalin** [reply]
Find out I can `import 'package:nylo_framework/nylo_framework.dart';` to be able to use `trans()` method.
